wamr-sdk/app/libc-builtin-sysroot/include/stdint.h

```diff
@@ -25,22 +25,44 @@ typedef unsigned long long int  uint64_t;
 typedef __INTPTR_TYPE__		intptr_t;
 typedef __UINTPTR_TYPE__	uintptr_t;
 
+/* Values to test for integral types holding `void *' pointer.  */
+#define INTPTR_MIN      INT32_MIN
+#define INTPTR_MAX      INT32_MAX
+#define UINTPTR_MAX     UINT32_MAX
```

Comment:
Had better add macro control, like L55
```C
#if UINTPTR_MAX == UINT64_MAX
#define INTPTR_MIN      INT64_MIN
#define INTPTR_MAX      INT64_MAX
#define UINTPTR_MAX     UINT64_MAX
#else
#define INTPTR_MIN      INT32_MIN
#define INTPTR_MAX      INT32_MAX
#define UINTPTR_MAX     UINT32_MAX
#endif
```

Reply:
I got it. These macros refer to the implementation of wasi, but I think your suggestion is correct.